app/marketing/mails.py

```diff
@@ -1237,34 +1237,13 @@ def new_bounty_daily(bounties, old_bounties, to_emails=None, featured_bounties=[
     if to_emails is None:
         to_emails = []
 
-    from marketing.views import quest_of_the_day, upcoming_grant, upcoming_hackathon, latest_activities, upcoming_dates, upcoming_dates, email_announcements
+    from marketing.views import quest_of_the_day, latest_activities, email_announcements
     quest = quest_of_the_day()
-    grant = upcoming_grant()
-    dates = list(upcoming_hackathon()) + list(upcoming_dates())
-    announcements = email_announcements()
-    chats_count = 0
 
     offers = f""
     if to_emails:
         offers = ""
-
-        profile = email_to_profile(to_emails[0])
-        chat = ""
-        if profile:
-            try:
-                chats_count = profile.chat_num_unread_msgs
-            except:
-                chats_count = 0
-            if chats_count:
-                plural = 's' if chats_count > 1 else ''
-                chat = f"💬 {chats_count} Chat{plural}"
-
-        notifications = get_notification_count(profile, 7, timezone.now())
-        if notifications:
-            plural = 's' if notifications > 1 else ''
-            notifications = f"🔵 {notifications} Notification{plural}"
-        else:
-            notifications = ''
```

Comment:
this will not remove notification email from townsquare?

Reply:
@jdorfman

Reply:
This PR only take care of the subject, by keeping it clean only using 💰Bounties, 🎯 Quest, ⚡️ New Action, & More (including quantities of each). So yeah the notifications subject is being taken out here

Reply:
@octavioamu @thelostone-mc this is for the Daily Digest. Town Square is in the email but removed from the subject. This was approved by @owocki